### ISSUE TITLE
FlowVN: Implement the tensorflow model

### ISSUE THREAD

**Issue by eryl** [NBISweden/aida-data-hub#534]
The model needs to be refactored. Currenly a lot of the legacy code is in the flowvn_tensorflow.py module, but all things pertaining to the training loop should be in the non-framework specific flowvn.py. This includes gradient accumulation, checkpointing, and so on.
